server.js

```diff
@@ -140,9 +146,9 @@ function new_client(id, opt, cb) {
 }
 
 module.exports = function(opt) {
-    opt = opt || {};
+    options = opt || {};
```

Comment:
`opt` only has scope in that function (by definition, being a function parameter. It wouldn't have been good enough to create a `var opt` in the parent scope (file)), but I needed something with scope that was accessible to the `maybe_bounce` function as well, so I created `options`.

There were different ways I guess I could have done this, like rename the parameter and then done `var opt;` on the other line you commented on.